spec/patches.md

```diff
@@ -80,33 +102,55 @@ Issue: The |taskQueue| in this step will either be a [=set=] of [=tasks=] or a [
 *roughly* compatible. Ideally, there would be a common task queue interface that supports a `pop()`
 method that would return a plain [=task=], but that would involve a fair amount of refactoring.
 
+### <a href="https://html.spec.whatwg.org/#queuing-tasks">Event Loop: Queuing Tasks</a> ### {#sec-patches-html-queuing-tasks}
+
+Change the <a href="">To queue a microtask</a> algorithm to accept an optional boolean
+|ignoreContinuationState| (default false).
+
+Change Step 5 to the following:
+
+  1. Let |continuationState| be |eventLoop|'s [=event loop/current continuation state=] if
+     |ignoreContinuationState| is false, otherwise null.
+  1. Set <var ignore=''>microtask</var>'s <a attribute for="task">steps</a> to the following:
+    1. If |ignoreContinuationState| is false, then set |eventLoop|'s
+       [=event loop/current continuation state=] to |continuationState|.
+    1. Run <var ignore=''>steps</var>.
+    1. If |ignoreContinuationState| is false, then set |eventLoop|'s
+       [=event loop/current continuation state=] to null.
+
 ### <a href="https://html.spec.whatwg.org/multipage/webappapis.html#hostmakejobcallback">HostMakeJobCallback(callable)</a> ### {#sec-patches-html-hostmakejobcallback}
 
 Add the following before step 5:
 
   1. Let |event loop| be <var ignore=''>incumbent settings<var>'s
      [=environment settings object/realm=]'s [=realm/agent=]'s [=agent/event loop=].
-  1. Let |state| be |event loop|'s [=event loop/current scheduling state=].
+  1. Let |state| be |event loop|'s [=event loop/current continuation state=].
 
 Modify step 5 to read:
 
  1. Return the <span>JobCallback Record</span> { \[[Callback]]: <var ignore=''>callable</var>,
     \[[HostDefined]]: { \[[IncumbentSettings]]: <var ignore=''>incumbent settings</var>,
     \[[ActiveScriptContext]]: <var ignore=''>script execution context</var>,
-    \[[SchedulingState]]: |state| } }.
+    \[[ContinuationState]]: |state| } }.
```

Comment:
Should the state be a copy of what is stored in the event loop? Otherwise the state might get changed before the job is run.

Reply:
Yeah that makes sense. In blink the state's properties are read-only (which would be nice to have in spec land), so copying makes sense. Heading out the door for a week, will pick this up when I get back.